prebid-server/pbs-modules/greenbids-analytics-reporter.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: page_v2
+page_type: pbs-module
+title: Prebid Server Greenbids PBS Analytics Reporter
+display_name : Prebid Server Greenbids Analytics Reporter
+sidebarType : 5
+---
+
+## Overview
+
+This analytics adapter communicates with Greenbids Analytics Server by sending to it the information about the SSP bids for the publisher inventory sold throguh PBS. For the publishers the incoming requests to the Greenbids Analytics Reporter require setup and approval from the Greenbids team. Please reach out to our team for more information [greenbids.ai](https://greenbids.ai).
+
+## Configuration
+
+Greenbids Analytics Reporter contains the static parameters defined in [application.yaml](https://github.com/EvgeniiMunin/prebid-server-java/blob/a2e376c14680108683483719d212d330f8f66742/src/main/resources/application.yaml#L296) 
+
+| Parameter | Type | Description |
+| --------- | ---- | ----------- |
+| analytics-server-version | string | Version of Analytics Schema Greenbids side |
+| analytics-server | string | Greenbids Analytics Server URL |
+| exploratory-sampling-split | float | Exploration vs Exploitation ratio of analytics traffic |
+| timeout-ms | int | Timeout limit on sending POST request to Greenbids Analytics Server |
+
+Greenbids Analytics Reporter also includes the parameters specific for each publisher setup. These parameters should be included into the extension of the bid request json: `ext.analytics.greenbids`
```

Comment:
Can this be changed to ext.**prebid**.analytics.greenbids? That's where https://docs.prebid.org/prebid-server/endpoints/openrtb2/pbs-endpoint-auction.html#analytics-adapter-flags suggests this kind of thing should go

Reply:
Thanks for the remark Bret! :pray:
We indeed use `ext.prebid.analytics.greenbids` , I have rechecked the [code](https://github.com/EvgeniiMunin/prebid-server-java/blob/adf9d76fb267c435d292281cf7bd5cd494d7cfdc/src/main/java/org/prebid/server/analytics/reporter/greenbids/GreenbidsAnalyticsReporter.java#L150-L154) of the parser and fixed the Doc